lib/checkMergeConflicts.js

```diff
@@ -108,3 +114,55 @@ module.exports.checkMergeConflictsInAllPullRequests = async function(context) {
     this.checkMergeConflictsInPullRequest(context, pullRequestDetails);
   }
 };
+
+/**
+ * This function checks whether a PR which requires merge to head for all
+ * other PRs is merged. If yes, it adds a comment and assigns authors on
+ * all PRs to do the same.
+ *
+ * @param {import('probot').Context} context
+ */
+module.exports.pingAllPullRequestsToMergeFromDevelop = async function (
+  context
+) {
+  /**
+   * @type {import('probot').Octokit.PullsGetResponse} pullRequest
+   */
+  const mergedPR = context.payload.pull_request;
+  const prLabels = mergedPR.labels.map((label) => label.name);
+  // Check if the PR that got merged requires all other PRs to be updated.
+  if (prLabels.includes(mergeAllPRsWithDevelopLabel)) {
+    // Fetch all open pull requests.
+    const pullRequestsResponse = await context.github.pulls.list(
+      context.repo({
+        per_page: 60,
+        state: 'open',
+      })
+    );
+
+    const allOpenPullRequests = pullRequestsResponse.data;
+    allOpenPullRequests.forEach(async (pullRequest) => {
+      // Comment on all pull requests.
+      const linkToChange = 'new change'.link(mergedPR.html_url)
+      const commentParams = context.repo({
+        body:
+          'Hi @' +
+          pullRequest.user.login +
+          ', there is a ' +
+          linkToChange +
+          ' in develop which needs to ' +
+          'be in your PR. Please update your branch with the latest changes ' +
+          'in develop, Thanks!',
```

Comment:
I would suggest update the wiki just before merge and add a small section on the label and how to merge your branch with develop (Similar to first instruction [here](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#address-review-comments-until-all-reviewers-give-lgtm-looks-good-to-me)). Then update this comment to include instructions on merging the branch with develop, will be helpful for new contributors. 

Just add one extra line line in comment, For instructions, refer this [Link to section you added].

Reply:
Can you please add link to that here as a pointer for new contributors who don't know about merging?